### PR TITLE
Modulos creados

### PR DESCRIPTION
**Pull request by mastalsbota**
La app con los módulos creados

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+ACmovies
```
Comment (jmanday) [inline .idea/.name:1]: We shouldn't upload the files related to the directory .idea/ and so one.

File: data/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (jmanday) [inline data/proguard-rules.pro:1]: It should be added to the gitignore file as well.

Comment (mastalsbota) [reply data/proguard-rules.pro:1]: We didn't talk about gitignore, imho we should have a meeting to make an agreement of the gitignore we want

Comment (melaniedura) [reply data/proguard-rules.pro:1]: I pretty much always use this template (will send it over Slack)

**Review by jmanday — APPROVED** [review]

**Review by melaniedura — APPROVED** [review]

**Review by Christianriverabalseras — APPROVED** [review]

**Pull request merged**